docs/source/ReferenceGuide.rst

```diff
@@ -24,6 +27,104 @@ results location::
 
   JOB LOG    : $HOME/avocado/job-results/job-2015-06-10T10.44-49ec339/job.log
 
+
+Test References
+---------------
+
+A Test Reference is a string that can be resolved into
+(interpreted as) one or more tests by the Avocado Test Resolver.
+Given resolver plugins are free to interpret a test reference,
+this string is completely abstract to the other components of
+Avocado.
+
+In the current implementation of Avocado, Test References can be
+provided explicitly in the command-line as parameters to 'avocado
+run' or implicitly via command-line switches like --vt-config
+from avocado-vt (which also works as a configuration mechanism of
```

Comment:
Clear examples are good, but since Avocado is a building block for Avocado-VT, and not the other way around, it *feels* wrong to point "that" way.